src/Paysera/WalletApi/Mapper.php

```diff
@@ -2393,4 +2407,61 @@ public function encodeTransferConfiguration(Paysera_WalletApi_Entity_TransferCon
     {
         return (new Paysera_WalletApi_Mapper_TransferConfigurationMapper())->mapFromEntity($transferConfiguration);
     }
+
+    /**
+     * @param array $clientPermissionsToWallets
+     * @return Paysera_WalletApi_Entity_ClientPermissionsToWallet[]
+     */
+    public function decodeClientPermissionsToWallets(array $clientPermissionsToWallets)
+    {
+        $result = [];
+
+        foreach ($clientPermissionsToWallets as $clientPermissionsToWallet) {
+            $result[] = $this->decodeClientPermissionsToWallet($clientPermissionsToWallet);
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param array $clientPermissionsToWallet
+     * @return Paysera_WalletApi_Entity_ClientPermissionsToWallet
+     */
+    public function decodeClientPermissionsToWallet(array $clientPermissionsToWallet)
+    {
+        return Paysera_WalletApi_Entity_ClientPermissionsToWallet::create()
```

Comment:
we should use **new Paysera_WalletApi_Entity_ClientPermissionsToWallet()**

Reply:
Done.